src/Microsoft.AspNetCore.Localization/RequestLocalizationOptions.cs

```diff
@@ -109,5 +109,53 @@ public RequestCulture DefaultRequestCulture
         /// </list>
         /// </summary>
         public IList<IRequestCultureProvider> RequestCultureProviders { get; set; }
+
+        
+        /// <summary>
+        /// Adds the set of the supported cultures by the application.
+        /// </summary>
+        /// <param name="cultures">The cultures to be added.</param>
+        /// <returns>The <see cref="RequestLocalizationOptions"/>.</returns>
+        public RequestLocalizationOptions AddSupportedCultures(params string[] cultures)
+        {
+            var supportedCultures = new List<CultureInfo>();
+
+            foreach (var culture in cultures)
+            {
+                supportedCultures.Add(new CultureInfo(culture));
+            }
+
+            SupportedCultures = supportedCultures;
+            return this;
+        }
+
+        /// <summary>
+        /// Adds the set of the supported UI cultures by the application.
+        /// </summary>
+        /// <param name="uiCultures">The UI cultures to be added.</param>
+        /// <returns>The <see cref="RequestLocalizationOptions"/>.</returns>
+        public RequestLocalizationOptions AddSupportedUICultures(params string[] uiCultures)
+        {
+            var supportedUICultures = new List<CultureInfo>();
+            foreach (var culture in uiCultures)
+            {
+                supportedUICultures.Add(new CultureInfo(culture));
+            }
+
+            SupportedUICultures = supportedUICultures;
+            return this;
+        }
+
```

Comment:
Your comment on the earlier bug included a method that added a list of cultures to both lists, any thoughts about that? Do you think that's important?

Reply:
Which one you mean?

Reply:
hmm. I looked again at the issue, and I couldn't find it. Ignore this comment I guess.